tests/wpunit.suite.dist.yml

```diff
@@ -21,6 +21,8 @@ modules:
             plugins:
                 - the-events-calendar/the-events-calendar.php
                 - event-tickets/event-tickets.php
+                - event-tickets-plus/event-tickets-plus.php
```

Comment:
I would not do this and burden a suite with an add-on requirement.

I would create a new `etplus_wpunit` suite, pretty much copy and paste the `wpunit` suite configuration files, and add tests in there.

Reply:
Done. Thanks. https://github.com/moderntribe/event-tickets/pull/1633/commits/0b393cc1738d206249dec3632d732e502c563d78